docs/fuchsia/setup.sh

```diff
@@ -78,7 +78,7 @@ build() {
     --with-base "//src/testing/fuzzing/syzkaller" \
     --args=syzkaller_dir="\"$syzkaller\"" \
     --variant=kasan
-  fx clean-build
+  fx build
```

Comment:
The actual code used by syzbot to build Fuchsia still uses clean-build:
https://github.com/google/syzkaller/blob/master/pkg/build/fuchsia.go#L53

`tools/syz-build` utility can be used to invoke the build procedure as it will be used by syzbot.

Reply:
That is interesting, we will look into it.
We are not sure whether we want syz-bot to use a clean build or incremental build (remember that at some point we even talked about downloading prebuilt images).

For local workflows, doing incremental builds is more convenient.